### PR TITLE
fix: unified graph captions.

### PR DESCRIPTION
**Pull request by emori-ctc**
## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py [hunk 1]
```diff
@@ -59,10 +59,11 @@ def create_figure(self) -> Figure:
         # # get stacked bar data
         data: dict[str, list[int | float]]
         y_labels: list[str] = []
-        y_axis_label = 'latency [ms]'
+        caption = 'latency'
+        y_axis_label = caption + ' [ms]'
         target_objects = self._metrics.target_objects
         data, y_labels = self._metrics.to_stacked_bar_data()
-        title: str = f"Stacked bar of '{getattr(target_objects, 'path_name')}'"
+        title: str = f'Stacked bar of  Response Time --- {self._case} case ---'
```
Comment (rokamu623) [inline src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py:66]: Together with latency, etc., it should be lower-cased.
```suggestion
        title: str = f'Stacked bar of  response time --- {self._case} case ---'
```

File: src/caret_analyze/plot/visualize_lib/bokeh/timeseries.py [hunk 2]
```diff
@@ -61,14 +62,8 @@ def create_figure(self) -> Figure:
...
-        if isinstance(target_objects[0], CallbackBase):
-            title = f'Time-line of callbacks {y_axis_label}'
-        elif isinstance(target_objects[0], Communication):
-            title = f'Time-line of communications {y_axis_label}'
-        elif isinstance(target_objects[0], Path):
-            title = f'Time-line of Paths {y_axis_label} --- {self._case} case ---'
-        else:
-            title = f'Time-line of publishes/subscribes {y_axis_label}'
+        title: str = f'Histogram of {data_type}'if self._case is None else\
+            f'histogram of {data_type} --- {self._case} case ---'
```
Comment (rokamu623) [inline src/caret_analyze/plot/visualize_lib/bokeh/timeseries.py:66]:
```suggestion
        title: str = f'Timeseries of {data_type}'if self._case is None else\
            f'Timeseries of {data_type} --- {self._case} case ---'
```

**Review by rokamu623 — CHANGES_REQUESTED** [review]
As shown below, the `hoge case` is also displayed when `create_latency_timeseries` is visualised.
bokeh_plot
The same applies for `create_frequency_timeseries` and `create_period_timeseries`.
Resolve this issue.

We also list two additional trivial mistake as follow:

File: src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py [hunk 3]
```diff
@@ -59,10 +59,11 @@ def create_figure(self) -> Figure:
         # # get stacked bar data
         data: dict[str, list[int | float]]
         y_labels: list[str] = []
-        y_axis_label = 'latency [ms]'
+        caption = 'latency'
+        y_axis_label = caption + ' [ms]'
         target_objects = self._metrics.target_objects
         data, y_labels = self._metrics.to_stacked_bar_data()
-        title: str = f"Stacked bar of '{getattr(target_objects, 'path_name')}'"
+        title: str = f'Stacked bar of response Time --- {self._case} case ---'
```
Comment (rokamu623) [inline src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py:66]: Not fixed.
```suggestion
        title: str = f'Stacked bar of response time --- {self._case} case ---'
```

**Review by rokamu623 — CHANGES_REQUESTED** [review]

File: src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py [hunk 4]
```diff
@@ -214,7 +214,8 @@ def histogram(
             raise NotImplementedError()
 
         plot: Figure = Figure(
-            title=data_type if case is None else f'{data_type} --- {case} case ---',
+            title=f'Histogram of {data_type}'
+            if case is None else f'histogram of {data_type} --- {case} case ---',
```
Comment (rokamu623) [inline src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py:218]:
```suggestion
            if case is None else f'Histogram of {data_type} --- {case} case ---',
```

**Review by rokamu623 — CHANGES_REQUESTED** [review]
Careless mistakes can still be found. Please fix.

File: src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py [hunk 5]
```diff
@@ -215,8 +215,9 @@ def histogram(
             raise NotImplementedError()
 
         plot: Figure = Figure(
-            title=data_type if case is None else f'{data_type} --- {case} case ---',
-            x_axis_label=x_label, y_axis_label='Probability', width=800
+            title=f'Histogram of {data_type}'
+            if case is None else f'Histogram of {data_type} --- {case} case ---',
+            x_axis_label=x_label, y_axis_label='Probability', plot_width=800
```
Comment (rokamu623) [inline src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py:220]:
```suggestion
            x_axis_label=x_label, y_axis_label='Probability', width=800
```

**Review by rokamu623 — CHANGES_REQUESTED** [review]
It is not compatible with bokeh 3.x.

**Review by rokamu623 — DISMISSED** [review]
LGTM!

**Review by rokamu623 — APPROVED** [review]
LGTM!

**Review by xygyo77 — APPROVED** [review]
LGTM
stacked_bar_bokeh_plot

**Pull request merged**